ci/docker/Dockerfile.build.jetson

```diff
@@ -31,6 +31,11 @@ ENV ARCH aarch64
 ENV HOSTCC gcc
 ENV TARGET ARMV8
 
+# gh issue #11567
```

Comment:
Please use absolute links as this makes it easier for us to pick them up using regex as well as making them one-clickable.

Reply:
done